src/utils.ts

```diff
@@ -1,3 +1,17 @@
 export function isWindowDefined() {
     return typeof window !== 'undefined';
 }
+
+export function removeCssClassName(element: HTMLImageElement | HTMLDivElement, cssClassName: string) {
+    element.className = element.className.replace(cssClassName, '');
+}
+
+export function addCssClassName(element: HTMLImageElement | HTMLDivElement, cssClassName: string) {
+    if (!element.className.includes(cssClassName)) {
+        element.className += `  ${cssClassName}`
+    }
+}
+
+export function hasCssClassName(element: HTMLImageElement | HTMLDivElement, cssClassName: string) {
+    return element.className && element.className.includes(cssClassName);
+}
```

Comment:
Even though these functions are quite simple, it should be nice to add some unit tests.

Reply:
We do not need to test `replace` and  `includes` but the code has logic that checks if `className` is defined and that line 11 only should run if `cssClassName` don't exist.